pages/level1/index.js

```diff
@@ -1,34 +1,158 @@
-import {useState} from "react";
+import {useEffect, useState} from "react";
 import Canvas from "../../components/Canvas";
 import {dataLevel1} from "../../components/Level1/_data";
-import Ressources from "../../components/Ressources/Ressources";
+import Ressources from "../../components/Ressources";
 import styled from "styled-components";
-import OverlaySmall from "../../components/OverlaySmall/OverlaySmall";
-import Header from "../../components/Header/Header";
+import OverlaySmall from "../../components/OverlaySmall";
+import Header from "../../components/Header";
+import BuildingMenu from "../../components/BuildingMenu";
+import cutTrees from "../../functions/cutTrees";
+import Tasks from "../../components/Tasks";
+import pumpWater from "../../functions/pumpWater";
 
 export default function Level1() {
   const [array, setArray] = useState(dataLevel1.fields);
+  const [buildingMenuState, setBuildingMenuState] = useState(false);
   const [overlayState, setOverlayState] = useState(true);
-  const text1 =
-    "Welcome to the island. Your first task is to build a lumberhut. Simply click on the sand tile, choose the lumberhut from the menu and build!";
+  const [textState, setTextState] = useState(1);
+  const [timeoutState, setTimeoutState] = useState(0);
+  const [pumpWaterState, setPumpWaterState] = useState(false);
+  const [possibleBuildings, setPossibleBuildings] = useState([
+    {name: "lumberhut", price: "wood x2"},
+    // {name: "house", price: ""},
+    //{name: "tower", price: "wood x4"}
```

Comment:
Please delete this unused lines of code.

Reply:
Those are notes for later use, i kinda use this code as a template for other levels....